Jenkinsfile

```diff
@@ -15,6 +15,11 @@
 // limitations under the License.
 // ------------------------------------------------------------------------------
 
+// Discard old builds after 31 days
+properties([[$class: 'BuildDiscarderProperty', strategy:
+        [$class: 'LogRotator', artifactDaysToKeepStr: '',
+        artifactNumToKeepStr: '', daysToKeepStr: '31', numToKeepStr: '']]]);
+
```

Comment:
Do you need to set artifactDaysToKeepStr too? The pipeline script generator also has a schedule/build periodically if not run option. Does leaving this option out mean the cleanup will only happen when the job is triggered by a PR?